htdocs/core/class/html.formcron.class.php

```diff
@@ -28,26 +28,6 @@
  */
 class FormCron extends Form
```

Comment:
But but ..., I didn't change that the extends is already existing it's why I removed the DB, ... because all these properties are already on the parent.

Anyway, all the formX classes are not forms all those methods are there to generate fields and not a form. One form is a collection of fields. Each of those fields should be in a delegate class much easier to manage. 

So, my idea is to keep those classes as today. But instead of introducing a class formX, it's the idea to introduce the form/ fields concept with namespaces. When we have that each help methods (if I can call them like that) will return the exact response as today (BC break) but based on the fields. And we have one file/class per field. Actually it's how it should be in POO.

If you can already accept this PR (which is in conflict now, I'll rebase) it would be appreciate.